vNext/src/Build.cmd

```diff
@@ -1,4 +1,4 @@
-@if "%_echo%" neq "on" echo off
+REM @if "%_echo%" neq "on" echo off
```

Comment:
lol, instead of this you can actually do:
`set _echo=on'
That was the purpose of that line (copied from corefx repo)